ffi/dotnet/Devolutions.IronRdp.ConnectExample/Devolutions.IronRdp.ConnectExample.csproj

```diff
@@ -10,5 +10,10 @@
   <ItemGroup>
     <ProjectReference Include="../Devolutions.IronRdp/Devolutions.IronRdp.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Windows.Compatibility" Version="8.0.3" />
+    <PackageReference Include="SixLabors.ImageSharp" Version="3.1.3" />
```

Comment:
question: Is it a leftover? If not, can look into using the standard library instead?

There are quite a few primitives available even in .NET core:
- https://learn.microsoft.com/en-us/dotnet/api/system.drawing?view=dotnet-plat-ext-8.0
- https://learn.microsoft.com/en-us/dotnet/api/system.drawing.image?view=dotnet-plat-ext-8.0
- https://learn.microsoft.com/en-us/dotnet/api/system.drawing.bitmap?view=dotnet-plat-ext-8.0

Reply:
updated